build.gradle

```diff
@@ -735,3 +736,88 @@ task distRpm(dependsOn: "jpackageImage", type: Rpm) {
+}
+
+task dockerStartOracleLinux8(type: DockerStartContainer) {
+	group = "RH8 RPM build container"
+	description = "Starts ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+}
+
+task dockerStopOracleLinux8(type: DockerStopContainer) {
+	group = "RH8 RPM build container"
+	description = "Stops ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+}
+
+task dockerRemoveOracleLinux8(type: DockerRemoveContainer) {
+	group = "RH8 RPM build container"
+	description = "Removes ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+}
+
+task dockerInstallYumOracleLinux8(type: DockerExecContainer) {
+	group = "RH8 RPM build container"
+	description = "Install git yum to ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+	commands = [["sh", "-c", "microdnf -y install git yum "] as String[]]
+}
+
+task dockerInstallDevToolsOracleLinux8(dependsOn: dockerInstallYumOracleLinux8, type: DockerExecContainer) {
+	group = "RH8 RPM build container"
+	description = "Install Development Tools to ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+	commands = [ ["sh", "-c", "yum -y groupinstall \"Development Tools\""] as String[] ]
+}
+
+task dockerClearCachesInLinux8( dependsOn: dockerInstallDevToolsOracleLinux8, type: DockerExecContainer) {
+	group = "RH8 RPM build container"
+	description = "Clear caches in ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+	user = "root"
+	commands = [ ["sh", "-c", "cd /source/anet ; rm -rf client/node_modules/; rm -rf /source/anet/build/*;" +
+			" rm -rf .gradle/caches/; rm -rf ~/.gradle/caches/; rm -rf /root/.gradle/caches ;"] as String[] ]
+}
+
+task dockerCopySource2BuildLinux8( dependsOn: dockerClearCachesInLinux8, type: DockerExecContainer){
+	group = "RH8 RPM build container"
+	description = "Generate /opt/anet_build folder and copy anet file in ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+	commands = [ ["sh", "-c", "rm -rf /opt/buildRpm/; mkdir /opt/buildRpm; cp -r /source/anet/ /opt/buildRpm/;"] as String[] ]
+}
+
+task dockerCleanGradleYarnCache(dependsOn: dockerCopySource2BuildLinux8, type: DockerExecContainer){
+	group = "RH8 RPM build container"
+	description = "Generate /opt/anet_build folder and copy anet file in ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+	commands = [ ["sh", "-c", " cd /opt/buildRpm/anet; ./gradlew yarn_cache_clean;  ./gradlew clean"] as String[] ]
+}
+
+task distRpmInDocker(dependsOn: dockerCleanGradleYarnCache, type: DockerExecContainer ) {
+	group = "RH8 RPM build container"
+	description = "Build ANET and Generate RPM  In ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+	commands =[ ["sh", "-c", "cd /opt/buildRpm/anet; ./gradlew distRpm; cp -r build/ /source/anet/; cd /source/anet/;" +
```

Comment:
This (`cp -r build/ /source/anet/;`) leaves files owned by `root` in my personal development directory.